### PR TITLE
Spec Doc Update

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

File: build/vega-lite-schema.json [hunk 1]
```diff
@@ -32,14 +32,15 @@
...
+                    "description": "An optional description of this mark for commenting purpose.",
                     "type": "string"
                 },
                 "encoding": {
                     "$ref": "#/definitions/EncodingWithFacet<Field>",
                     "description": "A key-value mapping between encoding channels and definition of fields."
                 },
                 "height": {
+                    "description": "Height of a single visualization.\n\n__Default value:__\n- For y-axis with a continuous (non-ordinal) scale, the height will be the value of [`config.cell.height`](config.html#cell-config).\n- For y-axis with an ordinal scale: if [`rangeStep`](scale.html#ordinal) is a numeric value (default), the height is determined by the value of `rangeStep` and the cardinality of the field mapped to y-channel.   Otherwise, if the `rangeStep` is `\"fit\"`, the height will be the value of [`config.cell.height`](config.html#cell-config).\n- If no field is mapped to `x` channel, the `height` will be the value of `rangeStep`.\n\n__Note__: For plot with `row` and `column` channels, this represents the height of a single cell.",
```
Comment (domoritz) [inline build/vega-lite-schema.json]: Cell config? nope.

File: src/spec.ts [hunk 2]
```diff
@@ -50,22 +49,40 @@ export interface BaseSpec {
   data?: Data;
 
   /**
-   * An object describing filter and new field calculation.
+   * An array describing data transforms such as filter and new field calculation.
```
Comment (domoritz) [inline src/spec.ts]: An Array of data transformations...

Comment (kanitw) [inline src/spec.ts]: I'll fix this in a follow up PR

Comment (kanitw) [inline src/spec.ts]: Don't wanna rebase blah blah right now

**Pull request merged**